src/main.ts

```diff
@@ -27,7 +33,7 @@ async function bootstrap(): Promise<{
   const swaggerConfig: SwaggerConfig = configService.get('swagger');
   const sentryConfig: SentryConfig = configService.get<SentryConfig>('sentry')!;
```

Comment:
all configs have to look like this

const sentryConfig: SentryConfig = configService.get<SentryConfig>('sentry');